### ISSUE TITLE
Core Tools 3.0.2852 does not print logs when ILogger<T> is used

### ISSUE THREAD

**Issue by zmarty** [Azure/azure-functions-core-tools#2184]
Azure Functions 3.0.14287.0
C# .Net Core 3.1
Azure Functions v34
Visual Studio 2019 16.7.2
Windows 10

Symptom: When we log using ILogger those log lines do not show up in func.exe console window anymore when debugging in Visual Studio.

Also opened here: https://github.com/Azure/azure-functions-host/issues/6582

**Comment by anthonychu** [comment]
@zmarty Gave it a try and ILogger lines are outputting for me. Can you share a bit more information about how you're writing the logs?

![image](https://user-images.githubusercontent.com/3982077/91918391-98f9ea00-ec77-11ea-9325-71ff1bfd891e.png)

/cc @pragnagopa

**Comment by pragnagopa** [comment]
@zmarty - please provide a repro and also details on which logs are missing.

Please see details in the https://github.com/Azure/azure-functions-core-tools/issues/2141 for recent changes we made to reduce system logging.

**Comment by zmarty** [comment]
I will send you an e-mail with the link to our source code (we are a Microsoft team). 

We use two versions of ILogger: One type of ILogger comes as argument or Azure Functions. The other (most common) type is a named ILogger<NameOfClass> which we request through dependency injection in the constructor of the classes which have Azure Functions.

In host.json we have the logging-logLevel set to Information. However, only Exceptions show up, which is not that useful for debugging and developing.

The bottom line is that the console used to show running logs and now it only shows Exceptions, and even those in a different (less useful) format than before. The only change was this new Azure Functions version.

**Comment by zmarty** [comment]
When I add _start --verbose_ under Debug > Application Arguments in Visual Studio and run a project I see the familiar debug information that used to show up as default from the Azure Functions host.

**However**, our own log lines still do not show up :(

**Comment by anthonychu** [comment]
@zmarty Thanks for sharing more information with us.

It appears that this is an issue when `ILogger<T>` is used. If anyone's running into this, please add a `default` log level in *host.json* and set it to `information` while we investigate this. For example:

```json
{
    "version": "2.0",
    "logging": {
        "applicationInsights": {
            "samplingExcludedTypes": "Request",
            "samplingSettings": {
                "isEnabled": true
            }
        },
        "logLevel": {
            "default": "Information"
        }
    }
}
```

**Comment by zmarty** [comment]
@anthonychu Thank you for the workaround. It works and we will use it for now.

Couple of things I noticed with this in place:
- I vaguely remember that log lines used to look like this in the past: [Information] Actual log Line      Am I right? I f yes, do you plan to bring back the log level label or it's gone for good?
- Settings logLevel default to Information also re-enables the infrastructure logging that was turned off (the one we normally need --verbose for). I am fine with this, just wanted to mention it. We actually would like a way to permanently set the output to verbose anyway, please think about having this as an option.

**Comment by zmarty** [comment]
Also, does setting logLevel default to Information change any of the logging done in production through Application Insights?

**Comment by pragnagopa** [comment]
>  logLevel default to Information change any of the logging done in production through Application Insights

Default level is `Information` - setting this explicitly should not change w.r.t. logs in Application Insights

**Comment by anthonychu** [comment]
@zmarty I think they used to show the timestamp.  See output [here](https://github.com/Azure/azure-functions-core-tools/issues/1131#issuecomment-653171101) (thread also has context on this change).

**Comment by zmarty** [comment]
@anthonychu  Yeah maybe. The problem now is that the log lines are an endless river of text, there is nothing separating one log line for another. Can you please add some marker or bring back the old functionality?

**Comment by pragnagopa** [comment]
Fix is merged and new version of core tools is released.

@zmarty - regading:

> The problem now is that the log lines are an endless river of text, there is nothing separating one log line from another. Some of our messages span multiple lines, making it even more difficult to parse by a human.

Please open a separate issue. Will discuss with @anthonychu
